### PR TITLE
Fixing error

### PR DESCRIPTION
**Pull request by physcocode**
Call to function like entities() and resultsIdentities() on Entity/Collection resulted in error "Undefined offset 0 on line 803"

### CODE REVIEW

File: lib/Query.php [hunk 1]
```diff
@@ -799,8 +799,10 @@ public function __call($method, $args)
 
             // Methods on Collection
         } elseif (method_exists('\\Spot\\Entity\\Collection', $method)) {
+            if(isset($args[0]))
```
Comment (vlucas) [inline lib/Query.php]: All `if` statements need to use curly braces and proper indentation. Fix this and I'll merge it :)

**Comment by physcocode** [comment]
@vlucas done !

**Comment by nini** [comment]
What if collection method has 2..n parameters on that?

**Comment by physcocode** [comment]
Well I din't encounter any issue during two parameter :/

**Comment by sebastian-galle** [comment]
additionally to @nini comments you can instead of checking the args[0] easily just cast it to array

`return call_user_func_array([$this->execute(), $method], (array) $args);`

and now you will be able to pass all args, for example for this method of Collection

`public function merge(Collection $collection, $onlyUnique = true)`

**Comment by vlucas** [comment]
I am fixing this for unlimited arguments via `call_user_func_array` now.

**Pull request merged**